docs/source/release/v6.0.0/sans.rst

```diff
@@ -20,12 +18,14 @@ Algorithms and instruments
 Improvements
 ############
 
- - In :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>`, the beam radius can be different for each distance.
-   A new parameter, TransmissionBeamRadius, has been added to set the beam radius for transmission experiments.
-   The default value of all beam radii is now 0.1m.
- - With :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>`, if sample thickness is set to -1, the algorithm will try to get it
-   from the nexus file.
- - With :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>`, the output workspace will get its title from the nexus file.
+- Added support for D11, D16, D22 and D33 in the :ref:`MaskBTP <algm-MaskBTP>` algorithm.
+- Several improvements have been done in ILL SANS suite :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>`:
+    - The beam radius can be different for each distance.
+    - A new parameter, TransmissionBeamRadius, has been added to set the beam radius for transmission measurements.
+    - The default value of all the beam radii is now 0.1m.
+    - If sample thickness is set to -1, the algorithm will try to get it from the nexus file itself.
+    - The output workspace will get its title from the nexus file.
+    - WavelengthRange is exposed to the algorithm, which is crucial for TOF reduction.
```

Comment:
Since the PR has been moved to release 6.1.0, should the release note be placed in v6.1.0 instead of v6.0.0?